pages/en/lb4/Creating-components.md

```diff
@@ -307,3 +307,296 @@ class EmailComponent {
   }
 }
 ```
+
+## Creating your own servers 
+
+LoopBack 4 has the concept of a Server, which you can use to create your own
+implementations of REST, SOAP, gRPC, MQTT and more. For an overview, see the
+[Server](#Server.html) section.
+
+### A Basic Example
+The concept of servers in LoopBack doesn't actually require listening on ports;
+you can use this pattern to create workers that process data, write logs, or
+whatever you'd like.
+
+Let's create an example server! Please note that this is not a complete example:
+
+```ts
+import {Server} from '@loopback/core';
+import {processReports} from '../util';
+export class ReportProcessingServer implements Server {
+  @inject('database') db: DefaultCrudRepository<Report, int>;
+  interval: NodeJS.Timer;
+  async start() {
+    // Run the report processor every 2 minutes.
+    let window = Date.now();
+    interval = setInterval(async () => {
+      const results = await db.find({
+        createdDate: {
+          $gt: window,
+        },
+      });
+      if (results && results.length > 0) {
+        results.forEach(async (rep) => {
+          await db.updateById(rep.id, await processReport(rep));
+        }); 
+      }
+      window = Date.now();
+    }, 1000 * 120);
+    return Promise.resolve();
+  }
+
+  async stop() {
+    clearInterval(interval);
+    return Promise.resolve();
+  }
+}
+```
+In this example, the `ReportProcessingServer` will load new reports every two
+minutes from the database (using the provided repository), process those entries
+and then update the database.
+
+### Using your server
+To use your server, simply bind it to your application's context.
```

Comment:
Do we want to call using methods like `.controller`, `.server`, etc. as bindings or does it make sense to say **To use your server, register it with your application via `.server()`**